_tour/self-types.md

```diff
@@ -24,12 +22,12 @@ trait User {
 }
 
 trait Tweeter {
-  this: User =>  // reassign this
-  def tweet(tweetText: String) = println(s"$username: $tweetText")
+	this: User =>  // reassign this
```

Comment:
Is there a reason this was changed from two spaces to a tab? I'd prefer keeping two-space indents.